### PR TITLE
TRACING-3922 | Configure OTEL to scrape in-cluster monitoring stack

### PR DESCRIPTION
**Pull request by pavolloffay**
Version(s):

4.12, 4.13, 4.14, 4.15, 4.16

This should be released as part of RHOSDT 3.2

Issue:

https://issues.redhat.com/browse/TRACING-3922

Link to docs preview:

https://71580--ocpdocs-pr.netlify.app/openshift-enterprise/latest/otel/otel-configuring#configuration-for-receiving-metrics-from-the-monitoring-stack_otel-configuring

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/otel-config-receive-metrics-monitoring-stack.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      pipelines:
+        metrics:
+          receivers: [prometheus]
+          processors: []
+          exporters: [debug]
+----
+<1> Give OpenTelemetry collector service account `cluster-monitoring-view` cluster role to access metric data.
+<2> Inject OpenShift service CA for configuring TLS in the Prometheus receiver.
+<3> Configure Prometheus receiver to scrape the federate endpoint from in-cluster monitoring stack.
+<4> Use the Prometheus query language to select metrics which should be scraped. Refer to the in-cluster monitoring documentation for more details and limitations of the federate endpoint.
```
Comment (max-cx) [inline modules/otel-config-receive-metrics-monitoring-stack.adoc:84]: @pavolloffay, would you be able to provide just a URL in reply in the comment here? I'll take care of the rest.
I tried searching the OpenShift docs and was not confident picking a link to add here.

Comment (pavolloffay) [reply modules/otel-config-receive-metrics-monitoring-stack.adoc:84]: I put a link to the monitoring docs in the `Additional resources` section 

```
* xref:../monitoring/accessing-third-party-monitoring-apis.adoc#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-third-party-monitoring-apis[Querying metrics by using the federation endpoint for Prometheus]
```

**Comment by max-cx** [comment]
@pavolloffay, you are welcome to close this PR. I opened https://github.com/openshift/openshift-docs/pull/73313, which has already been peer-reviewed.

**Pull request closed without merging**